packages/server/src/with-blitz.ts

```diff
@@ -77,6 +79,10 @@ export function withBlitz(nextConfig: any) {
 
         return config
       },
+      env: {
+        ...(normalizedConfig.env || {}),
+        NEXT_PUBLIC_REACT_MODE: experimental.reactMode,
```

Comment:
I don't think this needs to be prefixed with `NEXT_PUBLIC_`.  It could instead be `BLITZ_REACT_MODE`.  Environment variables set via the `env` key in config don't require the prefix; the prefix is only required for environment variables picked up from `.env.local`.

See https://nextjs.org/docs/api-reference/next.config.js/environment-variables and https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser